recursion/src/main/java/FactorialImpl.java

```diff
@@ -0,0 +1,9 @@
+public class FactorialImpl implements Factorial {
+
+  @Override
+  public int of(int base) {
+    return base != 0
```

Comment:
And negative `base`?

Reply:
Tests are green)

Reply:
ha! ) It's mirrored conversation in https://github.com/PaulFrmBrn/udemy-ds-and-algo-course/pull/6 )

Reply:
-7?
Can't get you.
I think we need to update tests - and after rebase I will add checks

Reply:
Ok, I'll do it

Reply:
https://github.com/PaulFrmBrn/udemy-ds-and-algo-course/pull/7